syncserver/src/web/middleware/rejectua.rs

```diff
@@ -37,7 +37,7 @@ pub fn reject_user_agent<B>(
           + 'static),
 ) -> LocalBoxFuture<'static, Result<ServiceResponse<EitherBody<B>>, actix_web::Error>> {
     match request.headers().get(USER_AGENT).cloned() {
-        Some(header) if header.to_str().map_or(false, should_reject) => Box::pin(async move {
```

Comment:
stupid nit, but I wonder if we should set `rust-version="1.70.0"` to indicate the base version we need? (might need higher)